pkgs/tools/networking/openssh/common.nix

```
@@ -28,11 +28,15 @@
 , libfido2
 , hostname
 , nixosTests
-, withFIDO ? stdenv.hostPlatform.isUnix && !stdenv.hostPlatform.isMusl
+, enableSecurityKey ? !stdenv.hostPlatform.isStatic
```

```suggestion
, withSecurityKey ? !stdenv.hostPlatform.isStatic
```
we should name things consistently